backend/config.py

```diff
@@ -11,6 +11,6 @@
 SWAGGERUI_BLUEPRINT = get_swaggerui_blueprint(SWAGGER_URL, API_URL, config={"app_name": "Python-Flask-REST"})
 
 DB_NAME = "covid-19"
-DB_HOST = "mongodb://mongodb_container:27017"
+DB_HOST = f"mongodb://{os.environ.get('DB_HOST', 'mongodb_container:27017')}"
```

Comment:
what about the passwrod and username to access the db? if we use some external db, for production we will not use docker for mongo

Reply:
for the live the DB_HOST could have the format: mongodb+srv://username:password@clusterid.sadasd.mongodb.net